### ISSUE TITLE
[Feature]: Add support for torchao quantification model

### ISSUE THREAD

**Issue by lessmore991** [vllm-project/vllm#10947]
### 🚀 The feature, motivation and pitch

When I use vllm to deploy a model quantified by the Torchao framework [https://huggingface.co/docs/transformers/v4.46.3/en/quantization/torchao#torchao], an error message shows that the quantization method is not supported.
` ...File "/usr/local/lib/python3.12/dist-packages/vllm/config.py", line 414, in _verify_quantization
    raise ValueError(
ValueError: Unknown quantization method: torchao. Must be one of ['aqlm', 'awq', 'deepspeedfp', 'tpu_int8', 'fp8', 'fbgemm_fp8', 'modelopt', 'marlin', 'gguf', 'gptq_marlin_24', 'gptq_marlin', 'awq_marlin', 'gptq', 'compressed-tensors', 'bitsandbytes', 'qqq', 'experts_int8', 'neuron_quant', 'ipex'].
`
Is there any plan to support torchao quantitative model in the future? Thanks

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by robertgshaw2-neuralmagic** [comment]
Not currently planned. Is there a certain scheme you want supported? We have https://github.com/vllm-project/llm-compressor for creating quantized models

**Comment by mapxin** [comment]
@robertgshaw2-neuralmagic 
Thank you, I will try to use llm-compressor. Does llm-compressor support the quantization of multi-modal models?

**Comment by simon-mo** [comment]
Yes. cc @mgoin

**Comment by mgoin** [comment]
Yes, @mapxin check out these FP8 quantization examples for [qwen2vl](https://github.com/vllm-project/llm-compressor/blob/1222bed98637c5aa628bad1b66b436b55ad73f95/examples/quantization_w8a8_fp8/qwen2vl_example.py) and [llama vision 3.2](https://github.com/vllm-project/llm-compressor/blob/1222bed98637c5aa628bad1b66b436b55ad73f95/examples/quantization_w8a8_fp8/llama3.2_vision_example.py).
